Problem139_ReconstructQueByHeight.java

```diff
@@ -0,0 +1,22 @@
+//Time Complexity: O(n log(n) + n) -> sorting + arranging
```

Comment:
Its actually O(n2) because we are arranging at the start in result list

`for(int [] ppl : people){
            result.add(ppl[1], ppl);
        }`

Reply:
got it! Thanks!